src/jaxsim/physics/model/physics_model_state.py

```diff
@@ -129,6 +129,16 @@ def valid(
 
 @jax_dataclasses.pytree_dataclass
 class PhysicsModelInput(JaxsimDataclass):
+    """
+    A class representing the input to a physics model.
+
+    This class stores the joint torques and external forces acting on the bodies of a physics model.
+
+    Attributes:
+        tau (jtp.Vector): An array representing the joint torques.
+        f_ext (jtp.Matrix): A matrix representing the external forces acting on the bodies of the physics model.
```

Comment:
Is it necessary to specify the attribute types? Having typing annotations, sphinx should detect them automatically, right?

```suggestion
    Attributes:
        tau: An array representing the joint torques.
        f_ext: A matrix representing the external forces acting on the bodies of the physics model.
```

Reply:
It might, but I made it explicit to match the rest of the code

Reply:
What are the classes in which docstrings report the explicit types of the attributes? I usually omit them.